src/Shell/Task/ExtractTask.php

```diff
@@ -352,6 +360,10 @@ public function getOptionParser()
             'boolean' => true,
             'default' => false,
             'help' => 'Do not write file locations for each extracted message.',
+        ])->addOption('no-marker-error', [
```

Comment:
It would be good to have a test for this option. The test could use `assertNotContains()` or `assertNotRegExp()` to ensure the error markers are not present.